src/MyParcelComApi.php

```diff
@@ -989,6 +994,47 @@ public function deleteShipmentSurcharge(ShipmentSurchargeInterface $shipmentSurc
         return true;
     }
 
+    /**
+     * @throws RequestException
+     * @throws JsonException
+     * @throws DateMalformedStringException
+     */
+    public function getDeliveryDates(
```

Comment:
Most (if not all) of the methods of this class extend the MyParcelComInterface. Let's add this one to the interface as well. Perhaps with a small description in the doc block that indicates what these delivery dates are for.